core/common/src/main/java/alluxio/wire/TieredIdentity.java

```diff
@@ -169,6 +170,41 @@ public String getTierName() {
     public String getValue() {
       return mValue;
     }
+
+    /**
+     * @param otherTier wire type locality tier to compare to
```

Comment:
Can you also add some unit tests to cover this method?

Reply:
Sure, I will add the test to the function.